dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -722,6 +722,101 @@
           "minimum": 1
         }
       }
+    },
+    "SummarizationTaskParametersBase": {
+      "type": "object",
+      "description": "Supported parameters for both Abstractive Summarization and Extractive Summarization task.",
+      "properties": {
+        "sentenceCount": {
+          "type": "integer",
+          "default": 3
+        },
+        "stringIndexType": {
+          "$ref": "#/definitions/StringIndexType"
+        }
```

Comment:
As noted in the other file, while the REST model will look the same, if these properties already GAd in 2022-05-01 (or whenever you last GA'd) they cannot be moved to another model. If necessary, you'll have to copy them everywhere.

Reply:
The same - nothing about summarization has GAed before.

Reply:
The same. I have copied the sentenceCount and stringIndexType to Abstractive task parameter and delete the base class.